desktop/core/src/desktop/settings.py

```diff
@@ -112,7 +112,7 @@
 # Examples: "http://media.lawrence.com/media/", "http://example.com/media/"
 MEDIA_URL = ''
 
-DATA_UPLOAD_MAX_MEMORY_SIZE = 5242880  # Setting this variable to 5MB as sometime request size > 2.5MB (default value)
+DATA_UPLOAD_MAX_MEMORY_SIZE = None
```

Comment:
Needs to a config, we had a escalation to support this.
Will there be a issue keeping default as unlimited. Also Add comment what does None mean, is it unlimited or something else.

Reply:
I added Django docs link for this field as a comment

Reply:
I don't think there is any harm in making this check as `None`. 
It just disables this check.